New version: HadronicLineshapes v0.4.1

JuliaRegistrator opened the pull request:
- Registering package: HadronicLineshapes
- Repository: https://github.com/mmikhasenko/HadronicLineshapes.jl
- Created by: @mmikhasenko
- Version: v0.4.1
- Commit: f4baa69c4790be4849ca7ee69344256928237915
- Reviewed by: @mmikhasenko
- Reference: https://github.com/mmikhasenko/HadronicLineshapes.jl/commit/f4baa69c4790be4849ca7ee69344256928237915#commitcomment-146175866
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd679b8065a5c574f171d771c6bc4534baec4fb9e381748a17ad1e7a86e906c98be61374dd932d22584a93ff41ed12394fa17f2c3cff50a4fc04dc1f421cc1fc34b04563eb3a6540858b3c780583fc61d027c6314ebc41bcad19167f12a3fd1f0bc32d2cdf74aa73f399998a6921b0002e5e087f3facd0f6bd153afb95f75cccaab96c73093aff164db257b95e01b0a312dde1160b3210a0214c1a4ba66b7f8f9f3ceb668906075badc3bb623d618dc461c9dda1afa765c13de628c92b9b01b84b7 -->

mmikhasenko commented:
#113488 is merged.
closing this PR, and opening a new one that won't be marked `new package`

JuliaRegistrator committed:
New version: HadronicLineshapes v0.4.1

UUID: 49c9d978-1f9d-4e96-a984-0a9783c0b9bf
Repo: https://github.com/mmikhasenko/HadronicLineshapes.jl.git
Tree: 7327537157a6cbd0231f3138aadf62ecbb44c4e0

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

mmikhasenko commented:
[no block]

@giordano thanks. Should be merged
